test/mocks/mocks.go

```diff
@@ -473,8 +473,8 @@ func ServiceBindingMock(
 			APIVersion: "operators.coreos.com/v1alpha1",
 		},
 		Spec: v1alpha1.ServiceBindingSpec{
-			MountPathPrefix: "/var/redhat",
-			CustomEnvVar:    []corev1.EnvVar{},
+			MountPath:    "/var/redhat",
```

Comment:
Remove this line

Reply:
Done https://github.com/redhat-developer/service-binding-operator/pull/735/commits/173e4233d503723751b0e24ec9f5bc6e428a4f29